bloom/generators/debian/generator.py

```diff
@@ -417,6 +417,19 @@ def generate_substitutions_from_package(
     data['debhelper_version'] = 7 if os_version in ['oneiric'] else 9
     # Summarize dependencies
     summarize_dependency_mapping(data, depends, build_depends, resolved_deps)
+    # Copyright
+    licenses = []
+    for l in package.licenses:
+        if hasattr(l, 'file') and l.file is not None:
+            license_file = os.path.join(os.path.dirname(package.filename), l.file)
+            if not os.path.exists(license_file):
+                error("License file '{}' is not found.".
+                      format(license_file), exit=True)
+            license_text = open(license_file, 'r').read()
+            if not license_text.endswith('\n'):
+                license_text += '\n'
+            licenses.append(license_text)
+    data['Copyright'] = '\n'.join(licenses)
```

Comment:
Do you have any in-the-wild example packages that you've tested this on? Depending on the contents of the individual license files a single newline between them may not be very readable.

I'm thinking of a case like
```
the final line of one license file.
Copyright 2015-2018 The authors of Another Project In The Package

The start of this second license text...
```